pkg/mtls/caprovider_secret.go

```diff
@@ -32,7 +32,7 @@ const (
 	clientCertCertKey   = "client.crt"
 
 	certOrganization       = "flotta-operator"
-	certRegisterCN         = "register"
+	CertRegisterCN         = "register"
```

Comment:
Why should this const be exported? Will it be used in the device?

Reply:
yes